scripts/analysis_1/nbconverted/3_visualize_data.py

```diff
@@ -223,10 +216,137 @@
 combined_data_df = pd.concat([input_data_UMAPencoded_df, simulated_data_UMAPencoded_df])
 
 # Plot
-ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=1) + facet_wrap('~dataset') + xlab('UMAP 1') + ylab('UMAP 2') + ggtitle('UMAP of original and simulated data')
+ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=1) + facet_wrap('~dataset') + xlab('UMAP 1') + ylab('UMAP 2') + ggtitle('UMAP of original and simulated data (gene space)')
+
+
+# In[27]:
+
+
+# Zoomed in view
+
+# Add label for input or simulated dataset
+input_data_UMAPencoded_df['dataset'] = 'original'
+simulated_data_UMAPencoded_df['dataset'] = 'simulated'
+
+# Concatenate input and simulated dataframes together
+combined_data_df = pd.concat([input_data_UMAPencoded_df, simulated_data_UMAPencoded_df])
+
+# Plot
+ggplot(combined_data_df, aes(x='1', y='2')) + geom_point(aes(color='color_by'), alpha=1) + facet_wrap('~dataset') + xlab('UMAP 1') + ylab('UMAP 2') + xlim(-2, 7) + ylim(-4, 7) + ggtitle('UMAP of original and simulated data (gene space)')
```

Comment:
Still hard to see if simulated data is representative of input...

Reply:
As you mentioned earlier, I think it would help users/readers to see the input data and the simulated data on the same plot but colored differently.